draft-ietf-tls-dtls13.md

```diff
@@ -486,6 +488,46 @@ protected portion.
 * Otherwise, the record MUST be rejected as if it had failed
 deprotection, as described in {{handling-invalid-records}}.
 
+{{demux}} shows this demultiplexing procedure graphically 
+taking DTLS 1.3 and earlier versions of DTLS into account. 
+~~~
+             +----------------+
+             | Decrypted      |
+             | Content Type   |
+             | (DCT)          |
+             |                |			 
+             |     DCT == 21 -+--> Alert
+       +---> |     DCT == 22 -+--> Handshake
+       |     |     DCT == 23 -+--> Application Data
+       |     |     DCT == 24 -+--> Heartbeat   
+       |     |     DCT == 26 -+--> ACK
+       |     |                |
+       |     +----------------+
+       |
+       |
+       |       Decryption
+       +-----------------------------------+
+                                           | 
+             +----------------+            |
+             | Outer Content  |            |
+             | Type (OCT)     |            |
+             |                |    /+------+-----+\
+packet  -->  | 31 < OCT < 64 -+--> DTLS Ciphertext
```

Comment:
The text above says you check this last, after the other types.

Reply:
I will re-order it.